debezium-core/src/test/java/io/debezium/junit/DatabaseVersionResolver.java

```diff
@@ -45,11 +45,11 @@ public boolean isLessThan(int major, int minor, int patch) {
                 return true;
             }
             else if (dbVersionMajor == major) {
-                if (minor != -1 && dbVersionMinor < minor) {
+                if (minor == -1 || (minor != -1 && dbVersionMinor < minor)) {
```

Comment:
```suggestion
                if (minor == -1 || dbVersionMinor < minor)) {
```

This should be enough? Same for the others?